### DISCUSSION TITLE
No icons showed in the 15.0.1 version

### DISCUSSION THREAD

**Discussion by ghost** [MurhafSousli/ngx-sharebuttons#687] [Q&A]
After migration to Angular 18 and ngx-sharebuttons 15 I cannot see the icons inside the buttons; my app cannot find the icon in the library; `Error: Could not find icon with iconName=facebook-f and prefix=fab in the icon library.`

The only solution I found was to follow the instructions for using custom FA icons; see  [https://ngx-sharebuttons.netlify.app/#/icons#custom-fa-icons](url)

Not found other ways to import the icons; tried aldo to follow the Icons Guide [https://ngx-sharebuttons.netlify.app/#/icons](url), but nothing changed.

Don't know if there's a bug  or the docs is very poor.

**Comment by MurhafSousli** [comment]
Please add a reproduction stackblitz?
